### DISCUSSION TITLE
Find the hidden number

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#935] [Galaxy Trivia] [answered]
Each row of equations is missing a digit that is repeated throughout the equation. Fill in the missing digit in the spaces provided, then fill in the shaded circle at the end of each row to reveal the hidden number.

<img width="275" alt="image" src="https://github.com/ParanoidUser/yolo/assets/5120290/3964885f-c373-4036-8466-bf6fa7f36123">

**Comment by YashNuhash** [comment, marked as answer]
> Question: Each row of equations is missing a digit that is repeated throughout the equation. Fill in the missing digit in the spaces provided, then fill in the shaded circle at the end of each row to reveal the hidden number.

 ## Here is my answer:

![image](https://github.com/ParanoidUser/yolo/assets/106915718/33582f14-d078-45aa-9f24-d61faaeb2a20)

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [Galaxy Brain](https://github.com/YashNuhash?tab=achievements&achievement=galaxy-brain) badge. Keep up the great work and unlock the Bronze 🥉 badge next by answering 8 questions. Don't forget to post your own questions too! 😊🎉

**Comment by peterbaccaro** [comment]
1. Missing 8 (58+88=146)
2. Missing 6 (64+61=125)
3. Missing 7 (73+78=151)
4. Missing 6 (56+61=117)
5. Missing 6 (65+63=128)
6. Missing 5 (52+85=137)
7. Missing 9 (95+39=134)
